settings/boss_build_settings.py

```diff
@@ -46,6 +46,7 @@ def create_settings(tmpl_fp, boss_fp):
     nd_config['aws']['cuboid_bucket'] = boss_config['aws']['cuboid_bucket']
     nd_config['aws']['tile_index_table'] = boss_config['aws']['tile-index-table']
     nd_config['aws']['cuboid_index_table'] = boss_config['aws']['s3-index-table']
+    nd_config['aws']['max_task_id_suffix'] = 100
```

Comment:
So this value needs to be kept in sync with the `boss-manage/lib/constants.py` value? Maybe a comments in both places so we know about the linkage, or have the value passed as part of boss_config.

Reply:
According to the top level file comment, this is only used for CI, but doing a grep, it might not even be used at all, anymore. Might be called as part of a Jenkins config, which isn't part of the repo. Just another reason to move to a CI solution that puts its config in the repo under test.